src/components/structures/auth/SetupEncryptionBody.js

```diff
@@ -132,32 +149,22 @@ export default class SetupEncryptionBody extends React.Component {
                 </AccessibleButton>;
             }
 
-            const brand = SdkConfig.get().brand;
+            let verifyButton;
+            if (store.hasDevicesToVerifyAgainst) {
+                verifyButton = <AccessibleButton kind="primary" onClick={this._onVerifyClick}>
+                    { _t("Verify against another session") }
```

Comment:
If all 3 buttons are shown, they seem to wrap to 2 lines, at least in Firefox:

<img width="613" alt="image" src="https://user-images.githubusercontent.com/279572/110329592-b9cf3d80-8014-11eb-9137-f4861a1339bf.png">

Reply:
fixed (and improved the wrap for other translations): 
<img width="627" alt="Screenshot 2021-03-08 at 17 59 17" src="https://user-images.githubusercontent.com/1294269/110361774-61aa3280-8038-11eb-8bbe-aebbea3d96f3.png">